script/dashboard/dashboard.js

```diff
@@ -157,3 +155,24 @@ app.controller('postcodeMapCtrl', function($scope, $http){
         });
     });
 });
+
+app.service('colorService', function() {
+    var o = {
+
+    };
+
+    o.getColor = function(ratio) {

```

Comment:
A racja :P Ale można prościej:
wywal:
var o = { } i return

i zmień na
this.getColor = fun...